.devcontainer/postCreate.sh

```diff
@@ -6,4 +6,8 @@ pip install poetry
 
 poetry install --with dev
 
-poetry run pre-commit install
\ No newline at end of file
+poetry run pre-commit install
+
+sudo apt-get update
+
+sudo apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libnss3 libxss1 libasound2 libxtst6 xauth xvfb
```

Comment:
This needs to go in the docker file and not here (also you shouldn't need sudo at that point)